dm/syncer/error.go

```diff
@@ -211,13 +212,66 @@ func (s *Syncer) handleSpecialDDLError(tctx *tcontext.Context, err error, ddls [
 	}
 	// TODO: add support for downstream alter pk without schema
 
+	// it handles the operations of DDL when encountering `invalid connection`
+	invalidConnF2 := func(tctx *tcontext.Context, err error, ddls []string, index int, conn *dbconn.DBConn, createTime int64) error {
```

Comment:
there are two ways to let `invalidConnF2` be able to use `createTime`, current code is adding a parameter, the other way is directly using `createTime` because `invalidConnF2` is a closure thus it can use implicit variables names in its surrounding scope.

It's OK for current code

Reply:
I see that other variables like `tctx`, `err`, `ddls`, `index`, `conn` are all passed through parameters, so here my thought is to conform with them by also passing `createTime` through parameter.